src/js/edu-benefits/utils/helpers.js

```diff
@@ -99,3 +99,25 @@ export function hasServiceBefore1978(data) {
 export function showRelinquishedEffectiveDate(benefitsRelinquished) {
   return benefitsRelinquished !== '' && benefitsRelinquished !== 'unknown';
 }
+
+export function getListOfBenefits(veteran) {
```

Comment:
Good question... I think this works for now and we can always add a TODO to revisit this later.